### ISSUE TITLE
fix or remove the ability to Reply to plaza posts

### ISSUE THREAD

**Issue by webgnar** [SkateHive/nextskateapp#17]
Replying on plaza doesnt seem to work. 
it opens a new page, which is not right.
it should expand a comment text field under the respective thing you are commenting on.
 allow you to type text, and upload media, and then nest the display of it, below it

**Comment by hunxjunedo** [comment]
Hey kindly assign this to me, would love to contribute if it's open to external contributions.

**Comment by sktbrd** [comment]
I think the best option will be a modal for creating a new reply and a dedicated page for posts on Feed that looks like twitter 

## Reply Modal Example: 

<img width="1231" alt="image" src="https://github.com/SkateHive/nextskateapp/assets/116202536/2a62f5ac-a486-4d0d-986d-dd9605b7866f">

## News post page example 

<img width="1231" alt="image" src="https://github.com/SkateHive/nextskateapp/assets/116202536/b539a050-4a8f-4620-9ebc-658fc3dbc4a8">

**Comment by sktbrd** [comment]
> Hey kindly assign this to me, would love to contribute if it's open to external contributions.

feel free to create pull requests , it will be reviewed and if merged you will me rewarded

**Comment by hunxjunedo** [comment]
Creating a pull request as soon as possible.

**Comment by hunxjunedo** [comment]
btw while working on this, I noticed the app is taking some 500mb ram, which aint normal. It needs some real optimization, which I intend to do in the future by opening a new issue, I know exactly why it's taking that much RAM.

**Comment by webgnar** [comment]
I opened a new issue for you called Optimize. Can you explain there why you think its using that much ram ?

**Comment by hunxjunedo** [comment]
I'm commenting on it, will decide which issue to resolve first.

**Comment by webgnar** [comment]
@sktbrd made some contributions to this before i officially assigned this to @hunxjunedo . the modal pops up now but does not work still. just letting you know ahead of time.
<img width="1284" alt="image" src="https://github.com/SkateHive/nextskateapp/assets/124617425/55e45fe9-9b49-414a-af83-b1ac9c46a5f2">

**Comment by hunxjunedo** [comment]
Will look into it

**Comment by hunxjunedo** [comment]
@sktbrd @webgnar just to be sure, what we intend to do is that when the user clicks on the comment icon, he should see a modal open instead of being redirected to another page right ?

**Comment by hunxjunedo** [comment]
does this look good ? 
![image](https://github.com/SkateHive/nextskateapp/assets/89797440/28e30e6d-957b-4068-895f-80e984678575)

**Comment by hunxjunedo** [comment]
> I think the best option will be a modal for creating a new reply and a dedicated page for posts on Feed that looks like twitter
> 
> ## Reply Modal Example:
> <img alt="image" width="1231" src="https://private-user-images.githubusercontent.com/116202536/333818597-2a62f5ac-a486-4d0d-986d-dd9605b7866f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3NTg0MDUsIm5iZiI6MTcxNjc1ODEwNSwicGF0aCI6Ii8xMTYyMDI1MzYvMzMzODE4NTk3LTJhNjJmNWFjLWE0ODYtNGQwZC05ODZkLWRkOTYwNWI3ODY2Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyNlQyMTE1MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yYTFiY2I4ZWUxYmM1ZmQ1MWQwY2RkNTI2NTMyMWQ0YTA1MGQ2MTkyNTE5OWRjOGQ5MWE4NjUyY2NjYjFlYWM4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.WhIayT0J3h-AxNmQoFXhUJdwcXPbTndU0eLAkjh4WDY">
> ## News post page example
> <img alt="image" width="1231" src="https://private-user-images.githubusercontent.com/116202536/333818629-b539a050-4a8f-4620-9ebc-658fc3dbc4a8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3NTg0MDUsIm5iZiI6MTcxNjc1ODEwNSwicGF0aCI6Ii8xMTYyMDI1MzYvMzMzODE4NjI5LWI1MzlhMDUwLTRhOGYtNDYyMC05ZWJjLTY1OGZjM2RiYzRhOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyNlQyMTE1MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMzgwZGMwNmM2YWM4OTc2NDE3YzcwMGFlYTJkOGRiNjM4YWQwMTI5ZTQzZGU0NWY5NDUxYWEwYzdjM2NkZWVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.4Ug1Bf2gnOiSHpNBZft6qtqeK9t9iChLOmW8iS3DEfQ">

also, I'm having trouble getting this: do we have to not show the comments ? because I just looked @webgnar 's modal, it only has the option to reply.

**Comment by hunxjunedo** [comment]
guys please confirm if the modal in the screenshot is ok so I can proceed with its functionality

**Comment by webgnar** [comment]
ok here is a clarification:
-it should look like [this](https://github.com/SkateHive/nextskateapp/issues/17#issuecomment-2132278179) (format)
-it should show the other replies too (my example did not have any)
- it can still display the value in $ usd that the post is at (small in green somewhere)
- replies can be replied to and will endlessly nest accordingly .
lmk if you have any other questions

**Comment by hunxjunedo** [comment]
> ok here is a clarification:
> -it should look like [this](https://github.com/SkateHive/nextskateapp/issues/17#issuecomment-2132278179) (format)
> -it should show the other replies too (my example did not have any)
> -it can still display the value in $ usd that the post is at (small in green somewhere)
> -replies can be replied to and will endlessly nest accordingly .
> 
> lmk if you have any other questions

So the example I sent is good to go right ?

**Comment by webgnar** [comment]
no. it does not look correct. update your branch and look at what we have so far. it just needs to be finished now

**Comment by hunxjunedo** [comment]
> no. it does not look correct. update your branch and look at what we have so far. it just needs to be finished now.
> look at the main branch which was just updated

I just checked it and it still doesn't show other replies. Should I go forward with my code as it's working as clarified by you ? Lmk if you have another idea.

**Comment by webgnar** [comment]
if you code shows other replies - like you showed, then yes we can use it. but your UI is not right so make sure you add it to our how we have it looking in the last update

**Comment by hunxjunedo** [comment]
So basically what I did was that convert the initial post page to a modal, so this modal has all the functionality that the post page had, including replying with photos etc.

**Comment by webgnar** [comment]
You are right. @sktbrd please stop and let @hunxjunedo finish before we style it. 
@hunxjunedo sorry about that. you can overwrite what he did or add to what he did. but we need it to look like how he has it looking.

**Comment by hunxjunedo** [comment]
Yeah I will try to make it look identical, coming back with a PR soon Inshallah. Thanks for understanding.

**Comment by hunxjunedo** [comment]
@sktbrd @webgnar let's close this issue today after the review, we have more work ahead.

**Comment by webgnar** [comment]
this still doesn't appear to work, so i cant close the issue.

**Comment by hunxjunedo** [comment]
I request maintainer access to this repository, I think this issue is taking unnecessarily long, I will right away resolve the conflicts and successfully merge it, I'm not new to this.
